docker-distribution.yaml

```diff
@@ -0,0 +1,144 @@
+package:
+  name: docker-distribution
```

Comment:
Just a nit, isn't the name here technically just distribution now that it moved to CNCF?

Reply:
yeah could be, good catch though!